$NetBSD: patch-configure,v 1.1 2021/05/17 10:38:48 micha Exp $

Add libtool support.
Upstream will not merge this (libtool is listed as not acceptable).

--- configure.orig	2021-05-04 18:40:48.000000000 +0000
+++ configure
@@ -3830,7 +3830,22 @@ if test "${enable_shared+set}" = set; th
   enableval=$enable_shared;
 fi
 
-if test x$enable_shared = xyes; then
+
+# Check whether --enable-libtool or --disable-libtool was given.
+if test "${enable_libtool+set}" = set; then
+  enableval="$enable_libtool"
+
+if test x$enable_libtool != xno; then
+    LIBTOOL="$enable_libtool"
+    enable_shared=no
+else
+    LIBTOOL=""
+fi
+fi
+
+
+if test "$enable_shared" = "yes"; then
+
     PICFLAG=1
     SHAREDSUFFIX=""
     FLUID="fluid-shared"
@@ -4071,6 +4086,20 @@ else
   FLUID_BUILD="../fluid/fluid$EXEEXT"
 fi
 
+if test "x$LIBTOOL" != "x"; then
+    LINKFLTK="../src/libfltk.la"
+    if test "x$LINKFLTKGL" != "x"; then
+        LINKFLTKGL="../src/libfltk_gl.la"
+    fi
+    if test "x$LINKFLTKIMG" != "x"; then
+        LINKFLTKIMG="../src/libfltk_images.la"
+    fi
+    LINKSHARED="-lfltk_images -lfltk_forms -lfltk"
+    DSONAME="libfltk.la"
+    FLDSONAME="libfltk_forms.la"
+    GLDSONAME="libfltk_gl.la"
+    IMGDSONAME="libfltk_images.la"
+fi
 
 
 
